qa-test-apps/helm-app/values-files-with-empty-values/README.md

```diff
@@ -0,0 +1,17 @@
+# Helm Example
+
+This example will deploy the [Kubernetes sample guestbook](https://github.com/kubernetes/examples/tree/master/guestbook/) application as
```

Comment:
Does this _need_ to be the guestbook? Could we use more lightweight charts instead (e.g. config map with or without service), which could speed up tests?

Reply:
could you please point me out, that is much appreciated.

Reply:
could you please point me out, that is much appreciated.

Reply:
Okay, I will update

Reply:
[this](https://github.com/rancher/fleet-test-data/tree/master/simple-chart) could be a good example.